### DISCUSSION TITLE
Support for multiple services with captcha

### DISCUSSION THREAD

**Discussion by BlackRandomCraft** [pterodactyl/panel#4702] [Feature Requests]
Add other captcha services along with reCAPTCHA, such as [hCAPTCHA](https://hcaptcha.com), [Cloudflare Turnstile](https://developers.cloudflare.com/turnstile/), [Yandex SmartCaptcha](https://cloud.yandex.com/en/services/smartcaptcha)

Where can I place it? In Admin -> Settings for example

**Comment by Molret** [comment]
nice feature.

**Comment by Otte-S** [comment]
Arix supports turnstile
